content/60.methods.md

```diff
@@ -16,37 +16,50 @@ In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becom
-Test CORD-19 statistics: {{cord19_total_pubs}} total publications.
+The contents of the COVID-19 Open Research Dataset (CORD-19) [@arxiv:2004.10706], which was developed in part to assist in efforts to train machine learning algorithms on COVID-19-related text, illustrates the volume of publication relevant to understanding this virus <!--(Figure @fig:cord19-growth).-->
+This resource was developed by querying several sources for terms related to SARS-CoV-2 and COVID-19, as well as the coronaviruses SARS-CoV-1 and MERS-CoV and their associated viruses [@arxiv:2004.10706] and contains {{cord19_total_pubs}} manuscripts as of {{cord19_date_pretty}}.
+Additional curation by CoronaCentral [@doi:10.1101/2020.12.21.423860] has produced, at present, a set of over 150,000 publications particularly relevant to COVID-19 and these closely related viruses.
+Thus, any effort to synthesize, summarize, and contextualize COVID-19 research will be faced with a vast corpus of potentially relevant material.
+
+<!--![
+**Change over time in the number of publications in the CORD-19 dataset.**
+As of {{cord19_date_pretty}}, there are {{cord19_total_pubs}} in the CORD-19 dataset.
+The first release, on March 16, 2020, contained 28,000 manuscripts on topics relevant to SARS-CoV-2 and related coronaviruses [@arxiv:2004.10706].
+Since then, the resource has continued to grow rapidly (left), with both traditionally published and preprint manuscripts in the corpus (right).
+At present, it contains {{cord19_total_preprints}} preprints from _arXiv_, _bioRxiv_, and _medRxiv_.
+While not all of the manuscripts are specifically focused on SARS-CoV-2 or COVID-19, this corpus is likely to contain all or most manuscripts that would be explored in developing a literature review, which requires assessing both emerging and prior research.
+]({{cord19-growth}} "Global COVID-19 deaths"){#fig:cord19-growth secno=1}
+-->
 <!--To Do: Possible viz of change in clinical trials as well?-->
 
-<!-- To Do: This is just an outline, formalize this paragraph with citations etc-->
-While some of this information has been disseminated by traditional publishing mechanisms, in other cases, it is made public through preprint servers or even press releases.
+With information being produced rapidly through both traditional publishing venues and preprint servers, some papers that are published face scrutiny after their initial release.
+Concerns have been raised that the number of COVID-19 papers being retracted may be higher, or potentially much higher, than is typical, although a thorough investigation of this question will not be possible until more time has elapsed [@doi:10.1080/08989621.2020.1782203; @doi:10.1080/08989621.2020.1793675].
+Other papers are updated with corrections or expressions of concern [@doi:10.1080/08989621.2020.1793675; @url:https://retractionwatch.com/retracted-coronavirus-covid-19-papers].
```

Comment:
Oh yikes, I had a few URLs in this one! I'm happy to just manually fix things if that's the easiest way to do it.

Reply:
I'm not sure what the best approach is.  Check the latest PDF build in https://github.com/greenelab/covid19-review/pull/943/#issuecomment-830197430 to see how these will render in the reference list.  Considering our time crunch, this may be low priority.  Let's take care of the other content first.